api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -159,6 +160,10 @@ public void validateInstitutionalAffiliation(Profile profile) {
           "Institutional role description cannot be empty when institutional role is set to Other");
     }
 
+    validateInstitutionalAffiliationAgainstEmail(profile);
+  }
+
+  private void validateInstitutionalAffiliationAgainstEmail(Profile profile) {
```

Comment:
broken out here so it can be called when either the VIA or the email change

Reply:
Can we just start calling it Affiliation? I.e. `validateEmailInAffiliatedDomain()`?

Reply:
Also, I try not to throw exceptions in services (especially exceptions intended for clients). Also, below, there's a 'validate' method that returns a boolean and doesn't throw:
```
!institutionService.validateAffiliation(dbVerifiedAffiliation, contactEmail)
```

Reply:
We throw all over this Service.  It's useful for me to do so here as well.

validateAffiliation() works for me